pkg/replicate/scheme.go

```diff
@@ -77,20 +77,28 @@ func (bf *BlockFilter) Filter(b *metadata.Meta) bool {
 	}
 
 	gotResolution := compact.ResolutionLevel(b.Thanos.Downsample.Resolution)
-	expectedResolution := bf.resolutionLevel
-
-	resolutionMatch := gotResolution == expectedResolution
+	resolutionMatch := false
+	for _, allowedResolution := range bf.resolutionLevels {
```

Comment:
What do you think about moving to map? :thinking:

Reply:
I hope I understood your intentions, is this what you had in mind? https://github.com/thanos-io/thanos/pull/2671/commits/76810cfb77c0d2da091a939a153805c5bd95d9a0